docs/quickstart.rst

```diff
@@ -0,0 +1,83 @@
+.. toctree::
+   :maxdepth: 1
+
+Quickstart
+==========
+
+Installation
+------------
+
+.. code-block:: bash
+
+    pip install rikai[torch]
+
+Use `Rikai` in the machine learning life cycle
+----------------------------------------------
+
+Lets get started from the feature engineering in `Spark`_.
+
+
+.. code-block:: python
+
+    from pyspark.ml.linalg import DenseMetrix
+    from rikai.types import Image, Box2d
+    from rikai import numpy as np
+
+    spark = (
+        SparkSession
+        .builder
+        .appName("rikai-quickstart")
+        .config("spark.jars.packages", "ai.eto:rikai:0.0.1")
+        .master("local[*]")
+        .getOrCreate()
+    )
+
+    df = spark.createDataFrame(
+        [
+            {
+                "id": 1,
+                "mat": DenseMatrix(2, 2, range(4)),
+                "image": Image("s3://foo/bar/1.png"),
```

Comment:
can Image handle http reference? Maybe we can make this work with an actual image?

Reply:
Do you have an actual image URL in mind?